tests/ProductTest.php

```diff
@@ -1674,6 +1675,29 @@ public function testSalesFrequencyIsProperlyExported(): void
         $this->assertSame(1337, $productMock->getField('sales_frequency'));
     }
 
+    public function testUrlIsBuiltFromFirstVariationIfMainVariationIsInactive(): void
+    {
+        $registryMock = $this->getRegistryMock(['get', 'getVatRateByVatId']);
+        $registryMock->expects($this->any())->method('get')->willReturnSelf();
+        $registryMock->expects($this->any())->method('getVatRateByVatId')->willReturn(1);
+
+        $productMock = $this->getProductMock(['getExportSalesFrequency', 'getRegistry', 'getStoreUrl', 'getItemId']);
+        $productMock->expects($this->once())->method('getExportSalesFrequency')->willReturn(true);
+        $productMock->expects($this->any())->method('getRegistry')->willReturn($registryMock);
+        $productMock->expects($this->any())->method('getStoreUrl')->willReturn('www.blub.io/');
+        $productMock->expects($this->any())->method('getItemId')->willReturn(69);
+        $productMock->setPath('/wohnzimmer/buro');
+
+        $response = $this->getMockResponse('/pim/variations/main_variation_inactive_other_is_active.json');
+        $productMock->processVariation($response['entries'][0]);
+
+        $this->assertEmpty($productMock->getField('url'));
+
+        $productMock->processVariation($response['entries'][1]);
+
+        $this->assertSame('http://www.blub.io//wohnzimmer/buro_69_1074', $productMock->getField('url'));
```

Comment:
Why double slash? Shouldn't the `trim` take care of that?

Reply:
That was actually an oversight of me. In the real world this does not happen, the test setup was just wrong at this point. I already fixed it before you sent the review ;).